HomeHealthServices/R4/files/HomeHealthServicesPrepopulation-0.0.1.cql

```diff
@@ -1,10 +1,68 @@
-library HomeHealthServicesPrepopulation version '0.0.1'
+library HomeHealthServicesPrepopulation  version '0.1.1'
 using FHIR version '4.0.0'
 include FHIRHelpers version '4.0.0' called FHIRHelpers
 
+// This cql and questionnaire combo can be considered a partial implementation of these forms:
+//  https://www.cms.gov/Outreach-and-Education/Medicare-Learning-Network-MLN/MLNProducts/Downloads/Medicare-Home-Health-Benefit-Text-Only.pdf
+
+codesystem "ICD-10-CM": 'http://hl7.org/fhir/sid/icd-10-cm'
+codesystem "LOINC": 'http://loinc.org'
+codesystem "SNOMED-CT": 'http://snomed.info/sct'
+codesystem "HCPCS": 'https://bluebutton.cms.gov/resources/codesystem/hcpcs'
+
+
+
 parameter service_request ServiceRequest
 
-context Patient
 
+context Patient 
+
+define "Today": Today()
+
+define "True": true
+
+define "False": false
+
+define RelevantDiagnoses: 
+  [Condition] condition
+  	where ('Condition/' + condition.id) = RequestEncounter.diagnosis[0].condition.reference.value
+
+define "RelavantDiagnosisCoding": ConvertConditionDetails(RelevantDiagnoses)
+
+define function ConvertConditionDetails(CondList List<Condition>):
+  distinct(
+    flatten(
+      CondList C
+        let ConditionCodings:
+          (C.code.coding) CODING
+          return FHIRHelpers.ToCode(CODING)     
+        return ConditionCodings
+    )
+  )
+
+define "OtherDiagnoses": 
+  ActiveOrRecurring([Condition] except RelevantDiagnoses)
```

Comment:
Since there is no valueset to filter out the condition, we should just use all patient's conditions in both relevant and other diagnoses (we discussed with Carmela during our HHS meeting). I have implemented in HHS-F2Fencounter-updated where I merged your branch to it. Please take a look. Thanks.